### ISSUE TITLE
Dependency on PyYAML < 6.0.0 prevents installation

### ISSUE THREAD

**Issue by fiapps** [pelican-plugins/pandoc-reader#33]
I'm trying to use this plugin in CI/CD to deploy a static site on push, but I can no longer build the environment to run `pelican` to build the site. It fails when it tries to use `pip` (and setuptools) to install `pyyaml < 6.0.0`, a requirement of `pelican-pandoc-reader` 1.2.0. `pyyaml` 5.4.1 fails to build due to its outdated `setup.cfg`. This problem is fixed in `pyyaml` 6.0.1, but I can't use that due to `pelican-pandoc-reader`'s requirement for an older version.

Here's a simplified Dockerfile that will reproduce the build failure.
```dockerfile
FROM pandoc/extra:3.1
RUN apk --no-cache add gcc musl-dev python3-dev
RUN pip install "pelican ~= 4.8" "pelican-pandoc-reader ~= 1.2"
```

The fix could be as easy as updating the requirement in `pyproject.toml`, but perhaps other changes will be needed for compatibility with PyYAML 6.0.1. I don't see any breaking changes in the [6.0.0 release notes](https://github.com/yaml/pyyaml/tags).

**Comment by nandac** [comment]
Thank you for alerting me to this I wil l submit a fix and make a new release.

**Comment by nandac** [comment]
@fiapps I have released version 2.0.0 and it is available now on PyPI so just upgrading the package is all that needs doing.

Kindly let me know if your problem has been fixed. Please note that this version drops support for Python 3.7 and the `PANDO_DEFAULT_FILES` variable in favour of the `PANDOC_DEFAULTS_FILES` variable.

**Comment by fiapps** [comment]
Thank you for fixing this quickly. My CI/CD is working again. Since I was already using `PANDOC_DEFAULTS_FILES`, all I had to do was update the version of `pelican-pandoc-reader` I use.
